js/ftx.js

```diff
@@ -473,35 +481,79 @@ module.exports = class ftx extends Exchange {
         //
         const result = [];
         const markets = this.safeValue (response, 'result', []);
+        this.oldSymbolMappings = {};
```

Comment:
We don't really need the oldSymbolMappings, i would drop that piece from this PR.
The old market ids will still work on the requesting side.

Reply:
I suggest that we just change the symbols and types in this case without looking into oldSymbolMappings.

Reply:
What if people were using symbols that look like 'ETH-PERP'?

Reply:
Tbh, I would just announce a switch, gather some feedback, and then would switch to new symbols from, say, 2022.
However, if you think that we should keep supporting them both in the long run, then we should probably unify the symbols-aliases in the base classes across the lib. Personally, I think just switching is fine, if the users are aware and prepared.

Reply:
On a second thought, we could change safeSymbol/safeMarket to account for a user-controllable property that would either return new symbols or old symbols. That would be cleaner, and we would not need the extra helper methods here and there.

Reply:
The old symbols for ftx are also the exchange ids, so those should work anyway shouldn't they?

Reply:
@samgermain yes, they should be ok.

Reply:
> @samgermain yes, they should be ok.

Ok, I remvoed `oldSymbolMappings`, still don't know what to do about `Trump2024` and `Bolsonaro2022` other than hard code them. They have expiry's of `Trump2024` and `Bolsonaro2022` right now in CCXT, but actual expiry's of oct 2024, and oct 2022